src/components/order/OrderFilter.js

```diff
@@ -71,7 +73,7 @@ class Filter extends PureComponent {
             this.currPageNum = pageNo;
             this.defaultPageUrl = 'pageNo=' + pageNo + 'size=' + this.size;
         }
-
+        this.setState({sortOrder: isAdmin(this.props.user) ? "+" : "-"});
```

Comment:
Setting ascending sort order for all types of status filter may not work in all cases. For example, in "All Orders" filters, it would be more likely that Admin user will need **most recent 25 orders** than **least recent 25 orders** (which would be older than a year or two!)